spec/models/option_set_import_spec.rb

```diff
@@ -19,7 +19,30 @@
 
     expect(option_set).to have_attributes(
       name: name,
-      geographic?: false)
+      geographic?: false,
+      is_standard: false)
+
+    expect(option_set.levels).to be_nil
+    expect(option_set.level_names).to include('en' => 'Province')
+
+    expect(option_set.total_options).to eq(26)
+    expect(option_set.all_options).to include(have_attributes(canonical_name: "Kinshasa"))
```

Comment:
This code should be moved into a helper method, like `expect_correct_option_set(option_set)` or something. Right now it's all repeated.